index.js

```diff
@@ -128,6 +128,12 @@ module.exports = {
     "no-undef": "off",
     "no-unused-vars": "off",
 
+    // emotion
+    "emotion/jsx-import": "error",
+    "emotion/no-vanilla": "error",
+    "emotion/import-from-emotion": "error",
+    "emotion/styled-import": "error",
```

Comment:
I'm wondering about the relevance of those rules, because in our case we want to import most of emotion related stuff from `sonar-ui-common/components/theme`. But those rules enforce that we import them from specific emotion packages so it goes against what we should really do. Maybe we should copy and customize those rules in this repo ? WDYT ?

Reply:
I agree, they're all about how/what is imported, but we only ever import it in `sonar-ui-common/components/theme`

Reply:
+1 for a custom rule.

Reply:
I agree too, but do we really need a custom rule to ensure we import things from `theme` ?

Reply:
I'd say yes if it's easy to have, otherwise we might need to care for emotion import in manual review